packages/terra-i18n/lib/translations/en-US.js

```diff
@@ -11,7 +11,12 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
 (0, _reactIntl.addLocaleData)(_en2.default);
 
 var messages = {
-  'Terra.ajax.error': 'This content failed to load.'
+  'Terra.ajax.error': 'This content failed to load.',
+  'Terra.demographicsBanner.dateOfBirth': 'DOB: ',
```

Comment:
Most of these translations will be concatenated with a value.  It is bad practice to concatenate translations: 
http://www.emreakkas.com/internationalization/10-internationalization-tips-for-developers-i18n-checklist

Instead, most of these translations should have a variable for value.